cid-redirects.json

```diff
@@ -2090,6 +2090,9 @@
   "/cid/9205": "/docs/search/behavior-insights/logexplain",
   "/cid/96734": "/docs/send-data/hosted-collectors/http-source/troubleshooting",
   "/cid/97652": "/docs/integrations/saas-cloud/qualys-vmdr",
+  "/cid/19877": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source",
+  "/cid/19878": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/airtable-source",
```

Comment:
This build failed because `/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/airtable-source` doesn't exist yet.

Reply:
Thanks Kim. `airtable-source` PR is merged, so it should work now.